src/scripts/mod.rs

```diff
@@ -0,0 +1,162 @@
+//! Scripting engine to run scripts based on tags and given ip-ports
+//! First read config file
+//!
+use anyhow::{anyhow, Result};
+use std::net::IpAddr;
+use std::path::PathBuf;
+use subprocess::{Exec, ExitStatus};
+use which;
+
+// struct ScriptEngine {
+//     nmap: Bin,
+//     python: Bin,
```

Comment:
We should allow them, in the config file, to specify a file path for Python too :) 

We're going to get a lot of comments like "Python doesn't work" when their default `python` is 2.7 and not `3.7`. Also, we'd need to make sure scripts are compatible with lower version of Python, or to have a version of python mentioned in the script name. 

So if the script tag is `python3.7` it doesn't run if the user has `3.6.9`

Reply:
Good idea, we can probably use absolute path, but if not we can change the working directory of RustScan on the fly and execute from the right directory with the right version of the interpreter.